orthoinference/src/main/java/org/reactome/orthoinference/Main.java

```diff
@@ -30,13 +29,15 @@ public static void main(String[] args) throws Exception {
 
 		Properties props = new Properties();
 		props.load(new FileInputStream(pathToConfig));
-        EventsInferrer.inferEvents(props, speciesCode);
-
-        // Link report file to report_ortho_inference.txt in website_files_update directory
-        Path pathtoWebsiteFilesUpdateWithReportFile = Paths.get(props.getProperty("pathToWebsiteFilesUpdateFolder") + "report_ortho_inference.txt");
-        Path reportFilename = Paths.get("report_ortho_inference_test_reactome_" + props.getProperty("releaseNumber") + ".txt");
-        Files.deleteIfExists(pathtoWebsiteFilesUpdateWithReportFile);
-        Files.createLink(pathtoWebsiteFilesUpdateWithReportFile, reportFilename);
+		EventsInferrer.inferEvents(props, speciesCode);
+
+		// Link report file to report_ortho_inference.txt in website_files_update directory
+		/*
+		Path pathtoWebsiteFilesUpdateWithReportFile = Paths.get(props.getProperty("pathToWebsiteFilesUpdateFolder") + "report_ortho_inference.txt");
+		Path reportFilename = Paths.get("report_ortho_inference_test_reactome_" + props.getProperty("releaseNumber") + ".txt");
+		Files.deleteIfExists(pathtoWebsiteFilesUpdateWithReportFile);
+		Files.createLink(pathtoWebsiteFilesUpdateWithReportFile, reportFilename);
+		*/
```

Comment:
Can this be removed?

Reply:
Yes, I just meant to keep it here as a reminder to implement the symlink code in Jenkins/bash. I'd like to keep it here and then maybe create a small PR in the future for that

Reply:
Sounds good.  Could you create an GitHub issue for that just so it stays on the task list radar?

Reply:
Done